docs/develop/update/network.rst

```diff
@@ -95,3 +95,20 @@ In the server's Pillar file, set ``network.netplan.template`` to ``custom`` and
                addresses:
                  - 198.51.100.34/32
                  ...
+
+NTP Servers
+-----------
+
+If the server is located outside of the UK you will need to manually configure NTP.
+
+You can do this by editing the server's Pillar file and configuring ``ntp`` to a list of the desired `NTP servers <https://www.pool.ntp.org/zone/europe>`__.
+
+For example to use the NTP servers located in Finland:
```

Comment:
Why choose one country or another? Do you choose the country in which the server is located?

You can clarify that the default is to use NTP servers located in the UK (the first paragraph wasn't clear on why servers within the UK don't need any change).

Reply:
Docs updated to clarify the above.